src/rules/indentation.js

```diff
@@ -42,7 +47,9 @@ function indentation(parsedFile, unused, configuration) {
 
     scenario.steps.forEach(function(step) {
       // Check Step indentation
-      test(step.location, configuration, 'Step');
+      var keyword = step.keyword.trim();
+      var type = keyword in availableConfigs ? keyword : 'Step';
```

Comment:
`'Given'`, `'When'` etc are defined in availableConfigs so `keyword in availableConfigs` will always evaluate true. I think we might want to keep a copy of the config that the user provided, plus the merged version of our defaults overlayed by the user config (that's what `configuration` is now) and use the user provided config to evaluate this line, and the merged configuration options for the rest of the checks that the rule does.